Exonoesis/Helping-Hand
# Main menu layout

## What is the purpose of these changes?
Most all games on the market have a title screen, from which the game can be started or the application exited at minimum. This can later be expanded to allow for changing settings, loading saved games, etc.

## What has changed?
The application now launches into a main menu with the game title and two buttons shown. Clicking the play button will load into the game and the application will function as before. Clicking the quit button will close the application.

## How can I verify this works?
1. Launch the application. Observe that the main menu now appears.
2. Click the button labeled "Quit." Observe that this closes the application.
3. Relaunch the application.
4. Click the button labeled "Play." Observe that you are now in the game as it was previously with no changes.

## Notes
Closes #17
